extension/lib/tracker.js

```diff
@@ -121,6 +125,7 @@ class Tracker {
                 return;
             }
             this.pointToContainer(this.containerId + 1); 
+            window.display.updateTimer(this.readableDomIds, this.containerId);
```

Comment:
I am leaning towards not having tracker not depend on display at all, and instead:
- [display, timer] do not know about each other's existence. They exist in isolation
- content_script.js is the main orchestrator that brings these indep components together
  - That means no need for window.display global variable.

Rationale:
- Being able to create independent, well-defined component that just does a few focused things very well makes it reusable by other future files. "O btw tracker also updates the timer display" is an extra dependency that will invite more future dependencies later on, and it will get harder to reason about what tracker.js actually does.
  -- doesn't really matter for our project I guess, but just a habit that i got used to.

In the end, up to you. Just a personal pref.

Reply:
I completely agree! I'll update in morning.